docs/Developer Resources/Building ZFS.rst

```diff
@@ -4,25 +4,24 @@ Building ZFS
 GitHub Repositories
 ~~~~~~~~~~~~~~~~~~~
 
-The official source for ZFS on Linux is maintained at GitHub by the
-`zfsonlinux <https://github.com/zfsonlinux/>`__ organization. The
-project consists of two primary git repositories named
-`spl <https://github.com/zfsonlinux/spl>`__ and
-`zfs <https://github.com/zfsonlinux/zfs>`__, both are required to build
-ZFS on Linux.
-
-**NOTE:** The SPL was merged in to the
-`zfs <https://github.com/zfsonlinux/zfs>`__ repository, the last major
-release with a separate SPL is ``0.7``.
-
--  **SPL**: The SPL is thin shim layer which is responsible for
-   implementing the fundamental interfaces required by OpenZFS. It's
-   this layer which allows OpenZFS to be used across multiple platforms.
+The official source for OpenZFS is maintained at GitHub by the
+`openzfs <https://github.com/openzfs/>`__ organization. The primary
+git repository for the project is the `zfs
+<https://github.com/openzfs/zfs>`__ repository.
+
+There are two main components in this repository:
 
--  **ZFS**: The ZFS repository contains a copy of the upstream OpenZFS
-   code which has been adapted and extended for Linux. The vast majority
-   of the core OpenZFS code is self-contained and can be used without
-   modification.
+- **ZFS**: The ZFS repository contains a copy of the upstream OpenZFS
+   code which has been adapted and extended for Linux and FreeBSD. The
+   vast majority of the core OpenZFS code is self-contained and can be
+   used without modification.
```

Comment:
```suggestion
- **ZFS**: The OpenZFS repository contains a generic code and system-dependent
   parts for Linux and FreeBSD. The vast majority of the core OpenZFS code
   is self-contained and can be used without modification.
```

Reply:
Thanks, wasn't sure what exactly was correct here. Typo: "contains generic code" instead of "contains a generic code"

Reply:
I like @rlaager 's wording more.